ui/ducks/send/send.js

```diff
@@ -1061,52 +1068,57 @@ const slice = createSlice({
       // input field
       state.recipient.userInput = action.payload;
     },
-    validateRecipientUserInput: (state, action) => {
+    updateRecipientWarning: (state, action) => {
+      state.recipient.warning = action.payload;
+    },
+    validateRecipientAddress: (state, action) => {
```

Comment:
[Q] I'm not against it just curious why this was changed? its been a while but i thought that the original intent was to validate the user's input *Before setting it as the actual address of the recipient in state*

Reply:
cc: @danjm

Reply:
➕

Reply:
It looks like you're setting recipient.address to `''` right before calling into here and then checking whether its `''`
on line [1642](https://github.com/MetaMask/metamask-extension/pull/13588/files#diff-bad76d7279418f3520a0ca93cfaeeb6fcbb1f0d4782f6ff942d422632a1c2780L1611)

Reply:
oh wait nvm thats in a reset method

Reply:
This appears to still require an answer.